docs/guides/admin/docs/workflowoperationhandlers/multiencode-woh.md

```diff
@@ -69,7 +69,7 @@ Each section runs independently as a parallel encoding job.
 For example, if presenter/source is to encoded with "mp4-low.http,mp4-medium.http" and
 presentation/source is to be encoded with "mp4-hd.http,mp4-hd.http"
 
-The target flavors are presenter/delivery and presentation/delivery and all are tagged "rss, archive".
+The target flavors are presenter/delivery and presentation/delivery and all are tagged "engage-streaming, example".
```

Comment:
Someone please make sure I'm not totally out to lunch here.  I *think* this is right?

Reply:
The wording for these paragraphs is fairly confusing and could be improved upon.

That being said, from my understanding (and testing) what you wrote is not correct. If we assume the `target-tags` from above (since no the text is not entirely clear on which tags it is talking about)

> <configuration key="target-tags">engage-streaming,example;engage-download,example</configuration>

, presenter-delivery should have "engage-streaming, example" and presenter-delivery should have "engage-download, example" (plus the encoding profiles). So the corrected version of your change would be.

> [...] and all are tagged "example".

Reply:
That I have trouble even parsing what the docs are saying speaks to a *dire* need to rewrite this whole thing.  I'll defer to your expertise here, and hopefully we can completely rewrite this Soon(tm)